### DISCUSSION TITLE
Question: timing on next release

### DISCUSSION THREAD

**Discussion by kgilmer** [i3/i3#5042] [Q&A]
Hello, there is at least [one fix](https://github.com/Airblader/i3/pull/434) I'd like to take in a future release of i3/i3-gaps into Regolith and am wondering if there is any expectation on when a `4.20.2` or `4.21` may be released?

**Comment by stapelberg** [comment]
My July is pretty full, so the earliest release date from my side would be in August. 

Looking at the changelog, most changes seem to be either fixes or maintenance work. I suppose it would be better to to make it 4.21, as we do bugfix release (like 4.20.2) only when they’re urgent, and only include absolutely-required fix commits in them.

**Reply by stapelberg** [reply]
See https://github.com/i3/i3/issues/5110 for the current blockers of the 4.21 release
